### DISCUSSION TITLE
In case of regret.... Will it go back to previous....

### DISCUSSION THREAD

**Discussion by Minglarn** [dwainscheeren/dwains-lovelace-dashboard#363] [Q&A]
Hi folks.. 
This look far to good to be unseen... :)
I'm eager to test this out, but let's say that i dont like it or it screws up my dasboards... 

Is it easy to go back to standard UI e.g. Home Assistant's? 
Or do I have to make a backup first?

**Comment by dwainscheeren** [comment]
DD will run stand-alone. So it won't touch any of your existing HA dashboards ui or yaml.

**Comment by dwainscheeren** [comment]
Hi,

I am closing the old Dwains Dashboard 3.x discussions as part of the DD3 cleanup.

DD3 is now in maintenance/community mode and is slowly being phased out. Active development is focused on Dwains Dashboard Next, the newest dashboard project:

https://github.com/dwainscheeren/dwains-dashboard-next

For chat, community help and discussion around DD3/DD Next, please join the Discord:

https://discord.gg/7yt64uX

If you still have a reproducible DD3 bug on the latest 3.9.x release, please open a new issue with current details. For new feature ideas, please try Dwains Dashboard Next.
